addon/components/hyper-table-v2/cell.ts

```diff
@@ -1,5 +1,37 @@
 import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import TableHandler from '@upfluence/hypertable/core/handler';
+import { Column, ResolvedRenderingComponent, Row } from '@upfluence/hypertable/core/interfaces';
 
-interface HyperTableV2CellArgs {}
+interface HyperTableV2CellArgs {
+  handler: TableHandler;
+  column: Column;
+  row: Row;
+  loading: boolean;
+}
 
-export default class HyperTableV2Cell extends Component<HyperTableV2CellArgs> {}
+export default class HyperTableV2Cell extends Component<HyperTableV2CellArgs> {
+  @tracked loadingCellComponent: boolean = false;
+  @tracked cellComponent?: ResolvedRenderingComponent;
+
+  constructor(owner: unknown, args: HyperTableV2CellArgs) {
+    super(owner, args);
+
+    this.loadingCellComponent = true;
```

Comment:
Nit: Same comment as on PR#110, can't you set this to true @line14 and remove this line ?